examples/shipments/buy-shipment.js

```diff
@@ -1,9 +1,10 @@
 const Easypost = require('@easypost/api');
 
-
 const api = new Easypost(process.env.API_KEY);
 
-api.Shipment.retrieve('shp_...').then(s => {
-  s.buy(s.lowestRate()).then(console.log).catch(console.log);
-  // s.buy('rate_...').then(console.log).catch(console.log);
-}).catch(console.log);
+api.Shipment.retrieve('shp_...')
+  .then((s) => {
+    s.buy(s.lowestRate()).then(console.log).catch(console.log);
+    // s.buy('rate_...').then(console.log).catch(console.log);
```

Comment:
can we remove this?

Reply:
Can you be more explicit about what you mean by `this`? It's an example showing you can either buy with the lowest rate or by passing in a rate ID. I won't be removing it as I believe it's useful and also not related to the changes made here, though we should go through and update all the examples separately (we have an internal task to do so).

Reply:
Sorry, I meant the comment in line 8: `// s.buy('rate_...').then(console.log).catch(console.log);`